src/main/java/seedu/address/model/task/Task.java

```diff
@@ -26,7 +26,7 @@
     private TaskType taskType;
     private RecurringType recurringType;
     
-    private List<TaskComponent> recurringDates;
+    public List<TaskComponent> recurringDates;
```

Comment:
This is not to access the list but to correct the reference.
It is being used in reset data for taskmaster